hx_lti_initializer/templates/hx_lti_initializer/instructor_index.html

```diff
@@ -0,0 +1,167 @@
+{% load static from staticfiles %}
+<html lang="en">
+<head>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js">
+    </script>
+    {# Load the tag library #}
+    {% load bootstrap3 %}
+
+    {# Load CSS and JavaScript #}
+    {% bootstrap_css %}
+    {% bootstrap_javascript %}
+
+    {# Display django.contrib.messages as Bootstrap alerts #}
+    {% bootstrap_messages %}
+    <meta charset="UTF-8">
+    <!-- TODO: Figure out how to get styles in there dynamically -->
+    <link rel="stylesheet" type="text/css" href="{% static 'hx_lti_initializer/instructor_styles.css' %}" />
+    <title></title>
+</head>
+<script>
+    jQuery(document).ready(function(){
+        jQuery('#assignments .clickable').click(function(event){
+            $(event.currentTarget).find('span').toggleClass('closeicon expandicon');
+        });
+    });
+    var printHelp = function(course_id, collection_id, object_id){
+        var el = '#copypaste-' + course_id;
+        jQuery(el).html("\
+            <p>\
+                In order to access the chosen object via LTI, make sure to add the following as custom parameters:\
+            </p>\
+            <pre>\
+            collection_id="+collection_id+"\
+            </pre>\
+            <pre>\
+            object_id="+object_id+"\
+            </pre>\
+        ");
+        jQuery(el).addClass('copypaste');
+    }
+</script>
+<div class="header">
+</div>
+<div class="container">
+    <div class="row">
+        <div class="col-md-3">
+            <div class="sidebar-top">
+                <div id="user-label">{{ user_name }}</div>
+            </div>
+            <div class="sidebar">
+                <p>
+                <ul>
+                    <li>To add a new course, simply initialize LTI tool as an administrator/instructor.</li>
+                    <li>Change course settings to edit the name and any people to be considered "admins"</li>
+                </ul>
+                </p>
+                <!--<div class="dashed-bar"></div>
+                <div class="dashed-bar"></div>
+                <p>
+                    <a class="link" href="./tutorial">View Tutorial</a>
+                </p>-->
+            </div>
+            <br/>
+            <a role="button" href="dashboard_view"><div class="btn btn-default">Instructor Dashboard</div></a>
+        </div>
+        <div class="col-md-1"></div>
+        <div class="col-md-8">
+            <div class="section-header">
+                Courses
+            </div>
+            {% for course in courses %}
+            <div class="course-name">
+                {{ course.course_name }}
+            </div>
+            {% if 'Administrator' in roles or 'Instructor' in roles %}
```

Comment:
It would be better to use symbols instead of string literals to check the roles. For example, this could create a bug if the "Administrator" role is actually "urn:lti:sysrole:ims/lis/Administrator". Unless you're certain that the administrator and/or instructor roles have been normalized in such a way that you can use string literals like that. Even then, it's better practice to use symbolic constants because that will protect against typos and other mistakes.